generators/firechip/src/main/scala/TargetConfigs.scala

```diff
@@ -59,26 +59,13 @@ class WithNIC extends icenet.WithIceNIC(inBufFlits = 8192, ctrlQueueDepth = 64)
 class WithNVDLALarge extends nvidia.blocks.dla.WithNVDLA("large")
 class WithNVDLASmall extends nvidia.blocks.dla.WithNVDLA("small")
 
-
-// Tweaks that are generally applied to all firesim configs
-class WithFireSimConfigTweaks extends Config(
+class WithFireSimDesignTweaks extends Config(
```

Comment:
Add a comment here explaining in greater detail what set of tweaks this contains.

Reply:
Wait Im actually a bit confused what you mean by this. There are already detailed comments next to each line on what things do.